layers.go

```diff
@@ -1431,7 +1413,7 @@ func (r *layerStore) Modified() (bool, error) {
 }
 
 func (r *layerStore) IsReadWrite() bool {
-	return r.lockfile.IsReadWrite()
+	return r.lockfile.IsReadWrite() && r.mountsLockfile != nil
```

Comment:
Please add a comment explaining the implications of `mountsLockfile`.

Reply:
Added comment as suggested :+1: